### PR TITLE
migrate to gh actions

### PR DESCRIPTION
**Pull request by eyenx**

### CODE REVIEW

File: .github/workflows/release-container-image.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      - 'v*.*.*'
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
```
Comment (tongpu) [inline .github/workflows/release-container-image.yaml:19]: Why is this action required?

**Review by tongpu — APPROVED** [review]
Looks good, I'm simply not sure why QEMU would be required.

File: .github/workflows/release-container-image.yaml (hunk 1 above)
Comment (tongpu) [reply .github/workflows/release-container-image.yaml:19]: Ah, I guess now I see it. It's required to build it for multiple architectures. Could you please add a comment explaining that.

Comment (hairmare) [reply .github/workflows/release-container-image.yaml:19]: I think Docker Buildx uses it. The job is how upstream documents we should use the action: https://github.com/docker/build-push-action#complete-workflow

**Comment by eyenx** [comment]
Is there really a need of a comment? it's taken from upstream.


-- 

Adfinis AG
Antonio Tauro, System Engineer, GPG KeyID: 0x0796132F0077A5F8
Güterstrasse 86 | CH-4053 Basel
Office +41 61 500 31 31 | Direct +41 61 500 31 37
www.adfinis.com

File: .github/workflows/release-container-image.yaml (hunk 1 above)
Comment (eyenx) [reply .github/workflows/release-container-image.yaml:19]: IMHO no comment needes, as it is how upstream documented it.

**Comment by eyenx** [comment]
It's taken from the example https://github.com/docker/build-push-action

I guess it's needed if you're building on a selection of platforms.

Toni



-- 

Adfinis AG
Antonio Tauro, System Engineer, GPG KeyID: 0x0796132F0077A5F8
Güterstrasse 86 | CH-4053 Basel
Office +41 61 500 31 31 | Direct +41 61 500 31 37
www.adfinis.com

**Pull request merged**